### PR TITLE
Added README file

### PR DESCRIPTION
**Pull request by Zel1oy**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,140 @@
-# This is my project
+# Book Store API
 
-Read file will be modified in the future
+Welcome to the Book Store API! This endeavor was ignited by the quest for a dynamic and effective solution to revolutionize the book store industry. Whether you're stewarding a brick-and-mortar haven for book lovers or curating an online literary emporium, our API is your steadfast companion, equipped to empower you with the tools essential for effortlessly managing your book inventory, organizing customer data, and orchestrating sales transactions with unmatched finesse.
+### Watch a [video](https://www.openai.com) about this project
```
Comment (andrii-hoienko) [inline README.md]: I thought that loom video should be here

File: README.md [hunk 2]
```diff
@@ -1,3 +1,140 @@
...
+
+   Before you begin, ensure you have the following prerequisites installed on your system:
+
+    - [Java 17](https://www.oracle.com/java/technologies/javase/jdk17-archive-downloads.html)
+    - [MySQL database](https://www.mysql.com/downloads/)
+    - [Docker](https://docs.docker.com/get-docker/)
+
+   ### Installation
+
+   Follow these steps to set up and run the Book Store API on your local machine:
```
Comment (andrii-hoienko) [inline README.md:76]: add alternative way to run app

**Review by andrii-hoienko — CHANGES_REQUESTED** [review]
GJ! Let`s improve your solution

File: README.md (hunk 2 above)
Comment (Zel1oy) [reply README.md:76]: I thought that AWS and locally enough, but tbh i have no idea what another way to add

Comment (andrii-hoienko) [reply README.md:76]: I mean without docker

**Review by andrii-hoienko — CHANGES_REQUESTED** [review]
Left minor comment

**Review by andrii-hoienko — APPROVED** [review]
GJ!

**Pull request merged**